scalableminds/webknossos
# Use webpack version 2

Uses webpack 2 for future proof bundling. Webpack 2 also has native ES2016 module support (not setup in this PR).

Steps to test:
- run `yarn install`
- `node_modules/.bin/webpack`
- do some light testing on webpack2.oxalis.at to see if all dependencies load properly
------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1626/create?referer=github" target="_blank">Log Time</a>
